### PR TITLE
fix conflicts

### PR DESCRIPTION
**Pull request by baxiry**

### CODE REVIEW

File: backend/db/manticore_db.py [hunk 1]
```diff
@@ -779,11 +792,46 @@ async def prepare_payload_doc(
...
+        for key in range(list):
+            try:
+                sql_str = "delete from key_value_pairs where key = '{key}'" 
+                self.utilsApi.sql(sql_str)
+            except:
+                return False
+        return True
+ 
+
     async def delete_doc_by_id(self, id: str) -> bool:
```
Comment (saadadel) [inline backend/db/manticore_db.py:814]: this function deletes the record from the corresponding table, use get_index_name_from_doc_id function to get the record index, and delete it from there

File: backend/db/manticore_db.py [hunk 2]
```diff
@@ -779,11 +792,46 @@ async def prepare_payload_doc(
...
+    async def delete_lock_doc(self, dto: EntityDTO) -> None:
+
+        docid = self.generate_doc_id(
+            dto.space_name, dto.branch_name, "lock", dto.shortname, dto.subpath
+        )
+
+        try:
+            await self.delete(key=docid)
+        except Exception as e:
+            logger.warning(f"Error at redis_services.delete_doc: {e}")
```
Comment (saadadel) [inline backend/db/manticore_db.py:832]: not **redis_services**

File: backend/db/manticore_db.py [hunk 3]
```diff
@@ -833,19 +881,88 @@ async def move(
...
+        )
+        lock_data = await self.get_lock_doc(
+            dto.space_name, dto.branch_name, dto.subpath, dto.payload_shortname
+        )
+        if not lock_data:
+            payload = {
+                "owner_shortname": owner_shortname,
+                "lock_time": str(datetime.now().isoformat()),
+            }
+            result = await self.save_doc(lock_doc_id, payload, nx=True)
```
Comment (saadadel) [inline backend/db/manticore_db.py:900]: where is this **save_doc** function defined?

File: backend/db/manticore_db.py [hunk 4]
```diff
@@ -833,19 +881,88 @@ async def move(
...
+            )
+            if lock_payload:
+                if dto.user_shortname:
+                    return bool(lock_payload["owner_shortname"] != dto.user_shortname)
+                else:
+                    return True
+            return False
+
+        except Exception as e:
+            logger.error(f"Error at BaseDB.is_entry_locked: {e.args}")
```
Comment (saadadel) [inline backend/db/manticore_db.py:951]: not ** BaseDB**

File: backend/db/manticore_db.py [hunk 5]
```diff
@@ -833,19 +881,88 @@ async def move(
...
-        pass
+
+        docid = self.generate_doc_id(
+            dto.space_name, dto.branch_name, "lock", dto.shortname, dto.subpath
+        )
+
+        try:
+            await self.delete(key=docid)
+        except Exception as e:
+            logger.warning(f"Error at redis_services.delete_doc: {e}")
```
Comment (saadadel) [inline backend/db/manticore_db.py:964]: not ** redis_services**

**Comment by baxiry** [comment]
my last change not appear !!

**Pull request closed without merging**